automation_oca/models/automation_record.py

```diff
@@ -89,56 +99,83 @@ def _search(
             count=False,
             access_rights_uid=access_rights_uid,
         )
-        if self.env.is_system():
-            # restrictions do not apply to group "Settings"
-            return len(ids) if count else ids
-
-        # TODO highlight orphaned records in UI:
-        #  - self.model + self.res_id are set
-        #  - self.record returns empty recordset
-        # Remark: self.record is @property, not field
 
         if not ids:
             return 0 if count else []
+
```

Comment:
Please don't add empty lines inside the method, specially if they weren't added previously